test/cli/autogen-pkg-autoloader/autogen-pkg-autoloader.out

```diff
@@ -140,6 +142,18 @@ require 'my_gem'
 
 Opus::Require.for_autoload(nil, "test/cli/autogen-pkg-autoloader/foo.rb", [RootPackage::Foo, :TOP_LEVEL_CONST])
 
+--- output/RootPackage/Nested.rb
+# frozen_string_literal: true
+# typed: true
+
+Opus::Require.on_autoload('RootPackage::Nested')
+
+module RootPackage::Nested
+end
+Opus::Require.register_package(RootPackage::Nested, 'RootPackage::Nested')
```

Comment:
Depending on what we want we could also make the 2nd arg something more like `RootPackage_Nested` or however we want to munge.

Reply:
Ah ok.